app/domain/authentication/authn_jwt/signing_key/fetch_provider_uri_signing_key.rb

```diff
@@ -0,0 +1,80 @@
+module Authentication
+  module AuthnJwt
+    module SigningKey
+      # This class is responsible for fetching JWK Set from provider-uri
+      class FetchProviderUriSigningKey < FetchSigningKeyInterface
+
+        def initialize(authentication_parameters:,
+          logger:,
+          fetch_required_secrets:,
+          resource_class:,
+          discover_identity_provider:)
+          @logger = logger
+
+          @fetch_required_secrets = fetch_required_secrets
+          @resource_class = resource_class
+          @resource_id = authentication_parameters.authn_jwt_variable_id
+          @discover_identity_provider = discover_identity_provider
+        end
+
+        def valid_configuration?
+          return @valid_configuration unless @valid_configuration.nil?
+          @valid_configuration ||= provider_uri_resource_exists?
+        end
+
+        def fetch_signing_key
+          discover_provider
+          fetch_provider_keys
+        end
+
+        private
+
+        def provider_uri_resource_exists?
+          !provider_uri_resource.nil?
+        end
+
+        def provider_uri_resource
+          @provider_uri_resource ||= resource
+        end
+
+        def resource
+          @logger.debug(LogMessages::Authentication::AuthnJwt::FetchingJwtConfigurationValue.new(provider_uri_resource_id))
+          @resource_class[provider_uri_resource_id]
+        end
```

Comment:
see this: https://github.com/cyberark/conjur/pull/2227/files?file-filters%5B%5D=.feature&file-filters%5B%5D=.rb&file-filters%5B%5D=.yml&file-filters%5B%5D=No+extension#r648240312

Reply:
I dont see anything in this link . Can you add screenshot

Reply:
it's a link to a comment above. please remove the `resource` method and instead log the message in the `provider_uri_resource` method

Reply:
Done